attributes/default.rb

```diff
@@ -25,27 +25,27 @@
 default['ndb']['group']                               = node['install']['user'].empty? ? "mysql" : node['install']['user']
 default['ndb']['connectstring']                       = ""
 
-default['ndb']['DataMemory']                          = "50"
-# Calculate IndexMemory size by default, can be overriden by user.
-default['ndb']['IndexMemory']                         = ""
+default['ndb']['DataMemory']                          = "98"
 default['ndb']['NoOfReplicas']                        = "1"
 default['ndb']['TcpBind_INADDR_ANY']                  = "FALSE"
 default['ndb']['NoOfFragmentLogParts']                = "4"
 default['ndb']['NoOfFragmentLogFiles']                = "4"
 default['ndb']['FragmentLogFileSize']                 = "64M"
-default['ndb']['MaxNoOfTables']                       = "3036"
+default['ndb']['MaxNoOfTables']                       = "4096"
 default['ndb']['MaxNoOfOrderedIndexes']               = "2048"
 default['ndb']['MaxNoOfUniqueHashIndexes']            = "512"
-default['ndb']['MaxDMLOperationsPerTransaction']      = "4297295"
+default['ndb']['MaxNoOfTriggers']                     = "768"
+default['ndb']['MaxDMLOperationsPerTransaction']      = "4294967295"
```

Comment:
This is not free. How much memory will be statically allocated for these resources?

Reply:
`4294967295` is actually the default value (https://dev.mysql.com/doc/refman/5.7/en/mysql-cluster-ndbd-definition.html#ndbparam-ndbd-maxdmloperationspertransaction) 
We don't set it on our cluster, also as far as I understood it's just a cap.
Also as we were setting it at `4297295` I assumed it was a typo.